src/cdk-experimental/menu/menu-item-trigger.ts

```diff
@@ -68,9 +68,11 @@ export function isClickInsideMenuOverlay(target: Element): boolean {
   host: {
     'class': 'cdk-menu-trigger',
     'aria-haspopup': 'menu',
+    '[attr.aria-expanded]': 'isOpen()',
+    '(focusin)': '_setHasFocus(true)',
+    '(focusout)': '_setHasFocus(false)',
```

Comment:
With Ivy the host bindings are inherited so you shouldn't have to copy these here.

Reply:
These are needed, they're on the trigger which does not inherit from `MenuBase`. The reason its needed here as well is because standalone menus don't have an inline menu for the user to tab into, instead they just have a button or something with a trigger on it